app/src/debug/res/layout/contactpatient_callpatient.xml

```diff
@@ -0,0 +1,348 @@
+
+  <TextView
+    android:id="@+id/helpTextView"
+    android:layout_width="@dimen/spacing_0"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="@dimen/spacing_16"
+    android:layout_marginTop="@dimen/spacing_32"
+    android:layout_marginEnd="@dimen/spacing_16"
+    android:layout_marginBottom="@dimen/spacing_8"
+    android:gravity="center|center_horizontal"
+    android:text="@string/contactpatient_help_text"
+    android:textAppearance="?attr/textAppearanceBody2"
+    android:textColor="@color/color_on_surface_67"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/removeFromOverdueListTextView"
+    tools:visibility="visible" />
+
+  <com.google.android.material.button.MaterialButton
+    android:id="@+id/normalCallButton"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="@dimen/spacing_16"
+    android:layout_marginEnd="@dimen/spacing_16"
+    android:paddingStart="@dimen/spacing_8"
+    android:paddingEnd="@dimen/spacing_24"
+    android:text="@string/contactpatient_call_normal"
+    android:theme="@style/ThemeOverlay.Simple.Blue2Primary"
+    app:icon="@drawable/ic_call_16dp"
+    app:layout_constraintEnd_toStartOf="@+id/secureCallButton"
+    app:layout_constraintHorizontal_chainStyle="spread"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/helpTextView"
+    tools:visibility="visible" />
+
+  <com.google.android.material.button.MaterialButton
+    android:id="@+id/secureCallButton"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:layout_marginEnd="@dimen/spacing_16"
+    android:paddingStart="@dimen/spacing_8"
+    android:paddingEnd="@dimen/spacing_24"
+    android:text="@string/contactpatient_call_secure"
+    app:icon="@drawable/ic_call_16dp"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toEndOf="@+id/normalCallButton"
+    app:layout_constraintTop_toTopOf="@id/normalCallButton"
+    tools:visibility="visible" />
+
+  <Space
+    android:id="@+id/spaceAboveNormalCallButton"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/spacing_12"
+    app:layout_constraintTop_toBottomOf="@id/normalCallButton" />
+
+  <RelativeLayout
+    android:id="@+id/noPhoneNumberBottomTextViewLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/spacing_32"
+    android:background="@color/simple_light_grey"
+    android:visibility="gone"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/removeFromOverdueListTextView">
+
+    <TextView
+      android:id="@+id/noPhoneNumberBottomTextView"
+      style="@style/TextAppearance.Simple.Button.Big"
```

Comment:
We are already applying text appearance for this text, so we don't need to directly reference the style here.